packages/web/src/pages/comment-history/components/desktop/CommentHistoryPage.tsx

```diff
@@ -139,6 +144,35 @@ const UserComment = ({ commentId }: { commentId: number }) => {
   )
 }
 
+const NoComments = ({ handle }: { handle?: string }) => {
+  const navigate = useNavigate()
+
+  const goToProfile = useCallback(() => {
+    if (handle) {
+      navigate(profilePage(handle))
+    }
+  }, [handle, navigate])
+
+  return (
+    <Flex column gap='2xl' p='l'>
+      <Flex column alignItems='center' gap='l'>
+        <IconMessage size='3xl' color='subdued' />
+        <Text variant='heading' size='s'>
+          {messages.nothingToDisplay}
+        </Text>
+        <Text variant='body' size='l'>
+          {messages.noComments}
+        </Text>
+      </Flex>
+      <Box>
+        <Button variant='secondary' size='small' onClick={goToProfile}>
```

Comment:
For buttons that act like links the move is:

<Button asChild ....><Link to={profilePage(handle)>{messages.backToProfile}</Link></Button>